build/views/index.html

```diff
@@ -23,6 +23,10 @@
       <li><strong>Out of hours</strong>: <em class="out-of-hours">{{(index .SupportRota "out-of-hours").Member}}</em></li>
       <li><strong>Escalations</strong>: <em class="escalations">{{(index .SupportRota "escalations").Member}}</em></li>
     </ul>
+    <button onclick="app.filterTeam('a')">Team A</button>
+    <button onclick="app.filterTeam('b')">Team B</button>
+    <button onclick="app.filterTeam('c')">Team C</button>
+    <button onclick="app.filterTeam()">Show all</button>
```

Comment:
I'm not a massive fan of the fact that `filterTeam` somewhat non-obviously takes an `undefined` `name` argument to mean "clear the filters". I think I'd rather there be a separate function called `clearTeamFilters()` that did this explicitly. I think this also would mean the implementation of `filterTeam` can be simplified.

Alternatively, if you like this kind of single-function-for-all-types-of-filtering thing, maybe `filterTeam` could take an anonymous function.

Reply:
I think that's even worse 😬.

Reply:
it was like that before I was asked to change it otherwise! :roll_eyes: 

:laughing: